publication-commons/src/main/java/no/unit/nva/publication/service/impl/UpdateResourceService.java

```diff
@@ -137,6 +139,17 @@ DeletePublicationStatusResponse updatePublishedStatusToDeleted(SortableIdentifie
         }
     }
 
+    ImportCandidate updateStatus(SortableIdentifier identifier, ImportStatus status)
+        throws NotFoundException {
+        var importCandidate = readResourceService.getResourceByIdentifier(identifier).toImportCandidate();
```

Comment:
is `readResourceService` a readonly service?

"resource" is a very generic name, does it hold anything?

Reply:
it can only read, yes